website/docs/r/workspace_run.html.markdown

```diff
@@ -0,0 +1,192 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_workspace_run"
+description: |-
+  Manages run create and destroy lifecycles in a workspace.
+---
+
+# tfe_workspace_run
+
+Provides a resource to manage create and destroy lifecycles in a workspace.
+The `tfe_workspace_run` expects to own exactly one apply during a creation and one destroy during a destruction. This implies that the `tfe_workspace_run` resource will queue an apply even though previous successful applies exist in the workspace prior to adding this resource.
```

Comment:
The wording here is a bit unclear.

Maybe something like:
> expects to own exactly one apply during a creation and/or one destroy during a destruction.

I.e. with both `apply` and `destroy` blocks being optional (but you have to specify at least 1), clarify the behaviour if you only specify one of them and not the other.